apps/api/src/usecase/podcast/generate_script_usecase.py

```
@@ -0,0 +1,165 @@
+        target_words = target_words or self.config.target_words
+
+        for attempt in range(self.config.max_retries_script_generation):
+            try:
+                return await self._generate_script_attempt(book_summary, book_title, target_words, include_intro_outro, language, attempt)
+            except PodcastScriptGenerationError:
+                raise
+            except Exception as e:
+                if attempt == self.config.max_retries_script_generation - 1:
+                    error_msg = f"Failed after {self.config.max_retries_script_generation} attempts: {str(e)}"
+                    if "finish_reason=10" in str(e) or "safety" in str(e).lower():
+                        error_msg += " (Content blocked by Gemini safety filters. The content may contain sensitive topics. Please try with different content.)"
```

Ruff (0.11.9)

Line too long (160 > 150)

(E501)